webapi/src/it/scala/org/knora/webapi/responders/admin/ProjectsResponderADMSpec.scala

```diff
@@ -370,24 +370,27 @@ class ProjectsResponderADMSpec extends CoreSpec with ImplicitSender {
       }
 
       "UPDATE a project" in {
+        val iri             = Iri.ProjectIri.make(newProjectIri.get).getOrElse(throw BadRequestException(""))
+        val updatedLongname = Name.make("updated project longname").getOrElse(throw BadRequestException(""))
+        val updatedDescription = ProjectDescription
+          .make(Seq(V2.StringLiteralV2("""updated project description with "quotes" and <html tags>""", Some("en"))))
+          .getOrElse(throw BadRequestException(""))
+        val updatedKeywords = Keywords.make(Seq("updated", "keywords")).getOrElse(throw BadRequestException(""))
+        val updatedLogo     = Logo.make("/fu/bar/baz-updated.jpg").getOrElse(throw BadRequestException(""))
+        val projectStatus   = ProjectStatus.make(true).getOrElse(throw BadRequestException(""))
+        val selfJoin        = ProjectSelfJoin.make(true).getOrElse(throw BadRequestException(""))
```

Comment:
Should it be empty `throw BadRequestException("")`?

Reply:
The `getOrElse` is only there to get the value out of the `Validation`. This is not supposed to fail in this test. But I agree that if something should fail in the future (because the definition / validation of the value objects changes), it would be more helpful to have meaningful error messages. I added them now.